exporter/pwd_file.go

```diff
@@ -0,0 +1,31 @@
+package exporter
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type PasswordMap struct {
+	RedisPwd map[string]string
+}
+
+func NewPasswordMap() *PasswordMap {
+	return &PasswordMap{}
+}
+
+func (p *PasswordMap) LoadPwdFile(pwdFile string) error {
```

Comment:
I think it made more sense what you had before. Don't think there's a need to make `LoadPwdFile` a member function of PasswordMap, just make it return `(*PasswordMap, error)`.